CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Every user-facing change should have an entry in this changelog. Please respect
 
 ## Unreleased
 
+## v14.0.5 (2022-08-09)
```

Comment:
@BbrSofiane Looks like the date needs to be updated here? (And presumably in the commit message, too?)

Reply:
I modified the date and merged the PR.